packages/docusaurus-theme-classic/src/theme/BlogArchivePage/index.tsx

```diff
@@ -28,7 +41,7 @@ function Year({year, posts}: YearProp) {
         {posts.map((post) => (
           <li key={post.metadata.date}>
             <Link to={post.metadata.permalink}>
-              {post.metadata.formattedDate} - {post.metadata.title}
+              {formattedDate(post.metadata.date)} - {post.metadata.title}
```

Comment:
This date should not have the year included, the goal is to fix https://github.com/facebook/docusaurus/issues/9830

The year is still displayed on the preview:
https://deploy-preview-9868--docusaurus-2.netlify.app/blog/archive

On this page I think it should be formatted with `year: undefined`